### PR TITLE
Replace mantid_loc with CONDA_ENV

### PR DESCRIPTION
**Pull request by jmborr**
References [[Task] Imp PD453](https://code.ornl.gov/sns-hfir-scse/diffraction/powder/powder-diffraction/-/issues/453) and [Update Conf file and Startup Script](https://code.ornl.gov/sns-hfir-scse/diffraction/powder/powder-diffraction/-/issues/454)

### CODE REVIEW

File: livereduce.spec [hunk 1]
```diff
@@ -20,6 +20,7 @@ BuildRequires: python%{python3_pkgversion} python%{python3_pkgversion}-setuptool
 
 Requires: python%{python3_pkgversion}
 Requires: python3-inotify
```
Comment (peterfpeterson) [inline livereduce.spec]: I think pyinotify can get removed since the conda environment provides that

File: scripts/which_python.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+import sys
+print("PYTHON_INTERPRETER =", sys.executable)
```
Comment (peterfpeterson) [inline scripts/which_python.py]: This should be moved to a log in `livereduce.py` to be something like (first message is in there already)
```python
logger.info('logging started by user \'' + os.environ['USER'] + '\'')
logger.info(f'using python interpreter {sys.executable}')
```
and this file be removed

**Review by peterfpeterson — CHANGES_REQUESTED** [review]
This also needs to bump the version number in the spec file

**Review by peterfpeterson — COMMENTED** [review]
This successfully makes an rpm with the correct contents. I have not tried to connect it to an instrument yet.

File: livereduce.spec [hunk 3]
```diff
@@ -19,7 +19,8 @@ Url: https://github.com/mantidproject/livereduce
 BuildRequires: python%{python3_pkgversion} python%{python3_pkgversion}-setuptools
 
 Requires: python%{python3_pkgversion}
-Requires: python3-inotify
+Requires: jq
+Requires: nsd-app-wrap
```
Comment (jmborr) [inline livereduce.spec:23]: I thought this `Requires` only applied to packages retrievable with `yum`. Is this only a declaration of requirements without any validation

**Review by jmborr — COMMENTED** [review]
looks good to me`

**Review by peterfpeterson — APPROVED** [review]

**Pull request merged**